Makefile

```diff
@@ -1,6 +1,6 @@
 # Set these to the desired values
 ARTIFACT_ID=k8s-dogu-operator
-VERSION=3.8.0
+VERSION=3.9.0
```

Comment:
We only update the version in release process. The right version ist 3.8.2. You have to integrate the current develop in your branch.